java/code/src/com/redhat/rhn/manager/content/ubuntu/UbuntuErrataManager.java

```diff
@@ -276,8 +276,20 @@ public static void processUbuntuErrata(Set<Channel> channels, List<Entry> ubuntu
                 errata.setAdvisoryType(ErrataFactory.ERRATA_TYPE_SECURITY);
                 errata.setIssueDate(Date.from(entry.getDate()));
                 errata.setUpdateDate(Date.from(entry.getDate()));
-                String[] split = entry.getId().split("-", 2);
-                errata.setAdvisoryRel(Long.parseLong(split[1]));
+
+
+                String[] split = entry.getId().split("-", 3);
+                if (split.length == 3 && split[0].equals("USN")) {
+                    errata.setAdvisoryRel(Long.parseLong(split[2]));
+                }
+                else if (split.length == 2) {
+                    errata.setAdvisoryRel(Long.parseLong(split[1]));
+                }
+                else {
+                    LOG.warn("Could not parse advisory id: " + entry.getId());
```

Comment:
This is adding a sonarlint issue here
```suggestion
                    LOG.warn("Could not parse advisory id: {}", entry.getId());
```